### PR TITLE
Dirt Bike Final Capstone

### PR DESCRIPTION
**Pull request by Tufahel**
> ###   Hello Reviewers  
> In this project, we have worked on `Book Dirt Bike app` is a motorcycle reservation platform that allows customers to book their motorcycle remotely within a defined time frame.
> 
> ### General requirements
> * [x]  Followed the **Github flow**.
> * [x]  There are no linter errors.
> * [x]  The documentation is professional.
> 
> # [PROJECT REQUIREMENTS](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md)
> 
> # [FRONTEND PULL REQUEST](https://github.com/Tufahel/book-dirt-bike-front-end/pull/10)
> # [LIVE SITE](https://incredible-unicorn-5f0c7d.netlify.app/)
> # [API DOCUMENTATION](https://calm-brushlands-14039.herokuapp.com/api-docs/index.html)
> Thank you for taking the time to review this pull request.

### CODE REVIEW

File: spec/requests/api/rentals_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'POST /api/rentals' do
+    it 'rentals posts successfully' do
+      post api_rentals_path
+      expect(response).to have_http_status(422)
+    end
+  end
```
Comment (Meltrust) [inline spec/requests/api/rentals_spec.rb]: - [ ] This test is failing for me, please, kindly consider correcting it to ensure the consistency of the app. 💪 

image

File: test/application_system_test_case.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'test_helper'
```
Comment (Meltrust) [inline test/application_system_test_case.rb]: - [ ] You are using Rspec, but this `test` folder is used by another test framework.  Please, consider deleting this folder if not being used.

File: README.md [hunk 3]
```diff
@@ -1 +1,99 @@
...
+Contributions, issues, and feature requests are welcome!
+
+## Credits
+
+Design inspiration: [Murat Korkmaz on Behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```
Comment (Meltrust) [inline README.md:118]: - [ ] Your project lacks a license file and a license section on your readme.  Please, kindly add the license file and corresponding readme section with a link to it. 

You can go [here]https://github.com/microverseinc/readme-template and grab the MIT.md file, put it at the root of your project, and link to it on your readme.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Tufahel ,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Nice design✔️
-  The app works ✔️ 
-  Good readme ✔️ 
-  Nice swagger ✔️ 

You can do it

## Suggested Changes ♻️
_Check the comments under the review._

You can use as many of my suggestions as you want.  If there is anything you would like to skip - feel free to do that.  However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: spec/requests/api/rentals_spec.rb (hunk 1 above)
Comment (Tufahel) [reply spec/requests/api/rentals_spec.rb]: Thank you very much for review @Meltrust . We have fixed this.
Screenshot from 2022-09-30 00-10-33

File: test/application_system_test_case.rb (hunk 2 above)
Comment (Tufahel) [reply test/application_system_test_case.rb]: We missed that, thanks for pointing out @Meltrust. We have fixed this.

File: README.md (hunk 3 above)
Comment (Tufahel) [reply README.md:118]: This is a very important element for a professional project. Thank you very much @Meltrust for suggesting this change. We have added it.

**Pull request merged**